### DISCUSSION TITLE
Use correct model for spawn dropships?

### DISCUSSION THREAD

**Discussion by Alystrasz** [R2Northstar/NorthstarMods#325] [Ideas]
I was looking for `PlayAnim` examples in NS code, when I found this:

https://github.com/R2Northstar/NorthstarMods/blob/31cba09fea290cfddde6cc77af836b1cd4756bdd/Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut#L109

Well, difference between two models is that `Goblins` are used by IMC, and `Crows` by Militia; so I suppose we could switch dropship model based on its team.

https://titanfall.fandom.com/wiki/Goblin
https://titanfall.fandom.com/wiki/Crow

**Comment by BobTheBob9** [comment]
is this actually the case in tf2? i'm relatively sure they both use the same model in tf2 since factions are almost exclusively a clientside thing

**Reply by ASpoonPlaysGames** [reply]
i believe so yeah, will try and find a clip

**Reply by ASpoonPlaysGames** [reply]
actually looking at the clips it seems that it doesnt depend on the faction, but it does change, presumably just one team uses one and the other team uses the other

**Reply by ASpoonPlaysGames** [reply]
![image](https://user-images.githubusercontent.com/66967891/165147509-051e2957-670c-4330-ba6f-4e8b39616a51.png)
![image](https://user-images.githubusercontent.com/66967891/165147615-4469bba8-f64e-4cb4-8171-d1960f45e8a8.png)

**Reply by ASpoonPlaysGames** [reply]
shit i fucked the screenshot

**Reply by ASpoonPlaysGames** [reply]
anyway, i assume that TEAM_MILITIA gets the crow, and TEAM_IMC gets the goblin
